backend/core/config.py

```diff
@@ -29,6 +31,36 @@
 app.include_router(chat_routes.router, tags=["chat"])
 
 
+@app.on_event("startup")
```

Comment:
@Galih, the pytest shows a warning after you added this on.event. Could you please try using synccontextmanager (from contextlib) and adding lifespan to the app. https://fastapi.tiangolo.com/advanced/events/

Reply:
I see, this alternative solution (on.event) is deprecated already (https://fastapi.tiangolo.com/advanced/events/#alternative-events-deprecated)